trunk/NDHMS/Data_Rec/module_RT_data.F

```diff
@@ -21,6 +21,7 @@
 Module module_RT_data
    use overland_data
    use module_reservoir_base
+   use module_read_timeslice_data
```

Comment:
If this is the only place this file is changed, I dont see that module_read_timeslice_data is actually needed.
In fact, I just compiled with that line commented out.

Reply:
Right. It can be removed. Thanks.

Reply:
@jdmattern-noaa fixing